cypress/integration/edit_team.spec.js

```diff
@@ -16,7 +16,7 @@ describe("Edit a team", () => {
 
     it("allows me to change the team's details", () => {
         cy.get("#f-name").clear().type("Another team");
-        cy.get("#f-teamType").select("COMPLAINTS");
+        cy.get("#f-teamType").select("ALLOCATIONS");
```

Comment:
This line is a bit pointless now since the select only has one option, but I see where that limitation comes from